src/lolex-src.js

```diff
@@ -245,11 +245,12 @@ function withGlobal(_global) {
         clock.timers[timer.id] = timer;
 
         if (addTimerReturnsObject) {
-            return {
+            var res = {
                 id: timer.id,
-                ref: NOOP,
-                unref: NOOP
+                ref: function () { return res; },
+                unref: function () { return res; }
```

Comment:
Can you add `.refresh` here?

Reply:
What is the `.refresh`?

Reply:
It's a new API Node recently added in https://github.com/nodejs/node/pull/20298 that landed in 10.2.0

Reply:
If you don't feel like it I'll land and can release when I get back from the Node summit/JSConf next week.

Just saying I would appreciate it if you did it while touching that code - this will land regardless - no pressure.

Already very happy that you decided to contribute to the project :)

Reply:
Ohh, I've not known about it. But I'm adding it.

Reply:
Done.

Reply:
I think adding [the whole logic](https://github.com/nodejs/node/pull/20298/files#diff-5e13dd343d32f1dc04d5b21db8979471R77) is a bit much in this (essentially unrelated PR), but adding a `NOOP` should be sufficient. Someone else can supply a full PR.